.changeset/wet-readers-join.md

```diff
@@ -0,0 +1,30 @@
+---
+'astro': minor
+---
+
+Shipped a new SSR mode, called `split`. 
+When enabled, Astro will emit a file for each page, which will render one single page.
+
+These files will be emitted inside `dist/pages`, and they will look like this:
+
+```
+├── pages
+│   ├── blog
+│   │   ├── entry._slug_.astro.mjs
+│   │   └── entry.about.astro.mjs
+│   └── entry.index.astro.mjs
+```
+
+To enable and customise this mode, new options are now available:
```

Comment:
```suggestion
To enable, set `build.split: true` in your Astro config:
```
I don't think this is customizable? Only enable-able? I would mention that this is a build option, one way or another here.